src/tests/functional/plugin/gpu/remote_blob_tests/gpu_remote_tensor_tests.cpp

```diff
@@ -435,12 +435,13 @@ TEST_P(OVRemoteTensor_TestsWithContext, smoke_canInferOnUserContextWithMultipleD
     // inference using remote tensor
 
     auto ocl_instance_tmp = std::make_shared<OpenCL>();
-    cl::Context multi_device_ctx({ocl_instance_tmp->_device, ocl_instance_tmp->_device});
+    auto ocl_instance_tmp_1 = std::make_shared<OpenCL>(1);
+    cl::Context multi_device_ctx({ocl_instance_tmp->_device, ocl_instance_tmp_1->_device});
     auto ocl_instance = std::make_shared<OpenCL>(multi_device_ctx.get());
 
     auto remote_context = ov::intel_gpu::ocl::ClContext(ie, ocl_instance->_context.get(), 1);
 
-    ASSERT_EQ(remote_context.get_device_name(), "GPU.0");
+    ASSERT_EQ(remote_context.get_device_name(), "GPU.1");
     auto exec_net_shared = ie.compile_model(function, remote_context, config);
```

Comment:
Hi @sshlyapn @vladimir-paramuzov , I want to test creating multiple context from user perspective, can  you help confirm if these change of multiple lines makes sense if I try to test GPU.1?
with these changes, I get below with the compile_model:
[ RUN      ] smoke_RemoteTensor/OVRemoteTensor_TestsWithContext.smoke_canInferOnUserContextWithMultipleDevices/RemoteTensor_Test_WITH_AUTO_BATCHING

MEM_USAGE=156456KB
unknown file: Failure
C++ exception with description "
/home/gta/bell/openvino/src/plugins/intel_gpu/src/plugin/program.cpp:389 cldnn program build failed! Kernel reorder_weights_opt_3_weight_0_0 not found in the kernel cache!" thrown in the test body.